verification/src/config.rs

```diff
@@ -31,13 +31,15 @@ impl Default for ServerConfiguration {
 pub struct SolidityConfiguration {
     pub enabled: bool,
     pub compilers_list_url: Url,
+    pub cron_schedule: String,
```

Comment:
Wouldn't be more indicative if we name it compilers_refresh_schedule?

Reply:
Also, do we want to allow users specify that no refreshing is required at all?

Reply:
how? it's not possible to make this field Option<String>, do you suggest to introduce new field like `compilers_refresh_enabled` ?